taskana-adapter-test/src/test/java/pro/taskana/adapter/integration/TestTaskAcquisition.java

```diff
@@ -0,0 +1,130 @@
+import pro.taskana.security.JAASRunner;
+import pro.taskana.security.WithAccessId;
+
+/**
+ * Test class to test the conversion of tasks generated by Camunda BPM to Taskana tasks.
+ * 
+ * @author Ben Fuernrohr
+ */
+@SpringBootTest(classes = TaskanaAdapterTestApplication.class, webEnvironment = WebEnvironment.DEFINED_PORT)
+@AutoConfigureWebTestClient
+@RunWith(JAASRunner.class)
+@ContextConfiguration
+public class TestTaskAcquisition extends AbsIntegrationTest {
+
+    @WithAccessId(
+        userName = "teamlead_1",
+        groupNames = {"admin"})
+    @Test
+    public void user_task_process_instance_started_in_camunda_via_rest_should_result_in_taskanaTask()
+        throws JSONException, InterruptedException {
+        String processInstanceId = this.camundaProcessengineRequester
+            .startCamundaProcessAndReturnId("simple_user_task_process", "");
+        List<String> camundaTaskIds = this.camundaProcessengineRequester
+            .getTaskIdsFromProcessInstanceId(processInstanceId);
+
+        Thread.sleep(this.adapterTaskPollingInterval);
+
+        for (String camundaTaskId : camundaTaskIds) {
+            List<TaskSummary> taskanaTasks = this.taskService.createTaskQuery().externalIdIn(camundaTaskId).list();
+            assertEquals(1, taskanaTasks.size());
+            String taskanaTaskExternalId = taskanaTasks.get(0).getExternalId();
+            assertEquals(taskanaTaskExternalId, camundaTaskId);
+        }
+    }
+
+    @WithAccessId(
+        userName = "teamlead_1",
+        groupNames = {"admin"})
+    @Test
+    public void multiple_user_task_process_instances_started_in_camunda_via_rest_should_result_in_multiple_taskanaTasks()
+        throws JSONException, InterruptedException {
+        int numberOfProcesses = 10;
+        List<List<String>> camundaTaskIdsList = new ArrayList<List<String>>();
+        for (int i = 0; i < numberOfProcesses; i++) {
+            String processInstanceId = this.camundaProcessengineRequester
+                .startCamundaProcessAndReturnId("simple_user_task_process", "");
+            camundaTaskIdsList
+                .add(this.camundaProcessengineRequester.getTaskIdsFromProcessInstanceId(processInstanceId));
+        }
+        Thread.sleep(this.adapterTaskPollingInterval);
+
+        for (List<String> camundaTaskIds : camundaTaskIdsList) {
+            for (String camundaTaskId : camundaTaskIds) {
+                List<TaskSummary> taskanaTasks = this.taskService.createTaskQuery().externalIdIn(camundaTaskId).list();
+                assertEquals(1, taskanaTasks.size());
+                String taskanaTaskExternalId = taskanaTasks.get(0).getExternalId();
+                assertEquals(taskanaTaskExternalId, camundaTaskId);
+            }
+        }
+    }
+
+    @WithAccessId(
+        userName = "teamlead_1",
+        groupNames = {"admin"})
+    @Test
+    @Ignore // variables are currently still stored in customAttributes, not the seperate columns
```

Comment:
It was never intended to put variables into the separate custom_x columns. These columns are reserved for customer defined fields... There may be a decision to put variables into another field (probably a new lob field) but until then, the current location is the 'correct' location. So please, change this testcase and check for the presence and correct value of the variables in customAttributes...

Reply:
The test now no longer references the custom_x columns but instead tests whether the attributes are integrated withing the customAttributes column